src/packages/recompose/renderNothing.js

```diff
@@ -1,9 +1,14 @@
+import { Component } from 'react'
 import createHelper from './createHelper'
 
-const renderNothing = _ => {
-  const Nothing = () => null
-  Nothing.displayName = 'Nothing'
-  return Nothing
+class Nothing extends Component {
+  render() {
+    return null
+  }
 }
```

Comment:
Yap, you're right. `UglifyJs` or other JS minifiers do not remove it. I have to admit that I haven't done this before. What I said is just from my limited knowledge :)